### PR TITLE
Fixes #21784: add a default value for syslog facility output for agent and server compontent

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/21784

### CODE REVIEW

File: webapp/sources/rudder/rudder-web/src/main/resources/rudder-system-global-parameter.conf [hunk 1]
```diff
@@ -8,7 +8,8 @@
     },
     "server": {
       "cf_serverd_bind_address": "::"
-    }
+    },
+    "syslog_facility": "LOG_LOCAL3"
```
Comment (amousset) [inline webapp/sources/rudder/rudder-web/src/main/resources/rudder-system-global-parameter.conf:12]:
```suggestion
    "agent_syslog_facility": "LOG_LOCAL3"
```

Comment (amousset) [reply webapp/sources/rudder/rudder-web/src/main/resources/rudder-system-global-parameter.conf:12]: IIRC it is only a setting for the agent

Comment (fanf) [reply webapp/sources/rudder/rudder-web/src/main/resources/rudder-system-global-parameter.conf:12]: if it's for agent only, it should be `"agent":  { "syslog_facility" : "LOG_LOCAL3" }`

If it's for both, it seems worthy to have a "log" category, it seems common enough to be useful. I'm not sure why but I would prefer to minimize number of top level props.

Comment (ncharles) [reply webapp/sources/rudder/rudder-web/src/main/resources/rudder-system-global-parameter.conf:12]: it's for both cf-agent and cf-serverd

Comment (fanf) [reply webapp/sources/rudder/rudder-web/src/main/resources/rudder-system-global-parameter.conf:12]: so, for now we will put it in log, and we will try to split / make it more granular if we want/can in further releases:
```
{ ...
"log": { "syslog_facility" : "LOG_LOCAL3" }
...
}
```

**Comment by ncharles** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-web/src/main/resources/rudder-system-global-parameter.conf (hunk 1 above)
Comment (ncharles) [reply webapp/sources/rudder/rudder-web/src/main/resources/rudder-system-global-parameter.conf]: Pr updated

**Review by fanf — APPROVED** [review]

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**